docs/setup_vscode_wsl.md

```diff
@@ -459,10 +426,24 @@ Run the program until it returns from the current function (or until the next br
 <img src="images/vscode_icon_continue.png" style="vertical-align: text-top; height: 1.25em;" /> **Continue**
 Run the program until the next breakpoint.
 
-### Example code
+#### Example code
+
+The examples below presume you have the sample `main.cpp` and `Makefile` files below.
+
+Download them from your terminal with:
+
+```console
+$ pwd
+/home/awdeorio/eecs280/p1-stats
+$ wget -nc https://eecs280staff.github.io/tutorials/main.cpp
+$ wget -nc https://eecs280staff.github.io/tutorials/Makefile
```

Comment:
If a student already has EECS 280 P1 starter files, with that mean they will run into a problem with the Makefile?  I think what would happen is that they will be missing a target for `main.exe`.

A possible solution would be to remove the `wget` parts and only have the "add code" instructions.

Reply:
We patched the P1 starter files to include a `main.exe` target in an earlier PR. But I do think we should remove that eventually.

The current instructions do tell them to copy in the target for `main.exe` if they have an existing `Makefile`.

Reply:
This is resolved via the other changes we discussed, having everyone use the sample files and introducing them earlier in the terminal.